### PR TITLE
Initialise accounting db for genesis block

### PR DESCRIPTION
**Pull request by azarovh**

### CODE REVIEW

File: chainstate/test-suite/src/tests/pos_processing_tests.rs [hunk 1]
```diff
@@ -1522,3 +1554,127 @@ fn decommission_from_not_best_block(#[case] seed: Seed) {
         res_pool_balance
     );
 }
+
+#[rstest]
+#[trace]
+#[case(Seed::from_entropy())]
+fn pos_stake_test_net_genesis(#[case] seed: Seed) {
```
Comment (TheQuantumPhysicist) [inline chainstate/test-suite/src/tests/pos_processing_tests.rs:1561]: testnet

File: chainstate/test-suite/src/tests/pos_processing_tests.rs [hunk 2]
```diff
@@ -1522,3 +1554,127 @@ fn decommission_from_not_best_block(#[case] seed: Seed) {
...
+    let upgrades = vec![
+        (
+            BlockHeight::new(0),
+            UpgradeVersion::ConsensusUpgrade(ConsensusUpgrade::IgnoreConsensus),
+        ),
+        (
+            BlockHeight::new(1),
+            UpgradeVersion::ConsensusUpgrade(ConsensusUpgrade::PoS {
+                initial_difficulty: MIN_DIFFICULTY.into(),
+                config: create_unittest_pos_config(),
```
Comment (TheQuantumPhysicist) [inline chainstate/test-suite/src/tests/pos_processing_tests.rs:1572]: This is wrong. We should start from testnet config... that's the whole point.

Comment (TheQuantumPhysicist) [reply chainstate/test-suite/src/tests/pos_processing_tests.rs:1572]: Please create separate config for testnet

**Review by TheQuantumPhysicist — APPROVED** [review]

**Comment by TheQuantumPhysicist** [comment]
This can be merged in the interest of getting wallet staking to work. The minor comments I mentioned can be addressed later.

**Review by alfiedotwtf — APPROVED** [review]

**Pull request merged**